stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1023,20 +1027,19 @@ extension String {
     scheme tagScheme: String,
     options opts: NSLinguisticTaggerOptions = [],
     orthography: NSOrthography? = nil,
-    tokenRanges: UnsafeMutablePointer<[Range<Index>]> = nil // FIXME:Can this be nil?
+    tokenRanges: UnsafeMutablePointer<[Range<Index>]>? = nil // FIXME:Can this be nil?
   ) -> [String] {
     var nsTokenRanges: NSArray? = nil
-    let result = tokenRanges._withBridgeObject(&nsTokenRanges) {
+    let op = {
```

Comment:
The three of us talked about this offline. I'm going to take another look at the `_withBridgeObject` cases again to see if I can simplify the control flow, and I'll put back the helper method (with a better name) if I can't.